src/commands.ts

```diff
@@ -594,6 +594,24 @@ export function registerCommands(context: vscode.ExtensionContext, reposManager:
 		});
 	}));
 
+	context.subscriptions.push(vscode.commands.registerCommand('pr.setFileListLayoutAsTree', async _ => {
+		vscode.workspace.getConfiguration('githubPullRequests').update('fileListLayout', 'tree');
+		await vscode.commands.executeCommand(
```

Comment:
We also need to set this value on startup, so that if you switch it to flat, the correct command is shown when you reload VS Code.

Somewhere in the `init` method in `extension.ts` would be a good place to put that